velodyne_pointcloud/include/velodyne_pointcloud/point_types.h

```diff
@@ -19,28 +19,39 @@
 #ifndef __VELODYNE_POINTCLOUD_POINT_TYPES_H
 #define __VELODYNE_POINTCLOUD_POINT_TYPES_H
 
-#include <pcl/point_types.h>
+#include <sensor_msgs/PointCloud2.h>
+#include <sensor_msgs/point_cloud2_iterator.h>
 
 namespace velodyne_pointcloud
 {
   /** Euclidean Velodyne coordinate, including intensity and ring number. */
-  struct PointXYZIR
+/*  struct PointXYZIR
```

Comment:
Do we know which packages ? I cannot tell from http://rosindex.github.io/p/velodyne_pointcloud/github-ros-drivers-velodyne/#indigo-deps (there should be some PR2 stuff in there no ?)

The packing of that struct is using PCL which means that to read it, you also need PCL. Maybe a break of compatibility using that PR would help in making that format be easier to read/write.